plugins/services/src/js/constants/OperatorTypes.js

```diff
@@ -1,10 +1,28 @@
 const OperatorTypes = {
```

Comment:
It provides types and info whether they require value or not. I think the file name is ok.